README.md

```diff
@@ -1,6 +1,6 @@
 # Office Add-ins documentation
 
-Welcome to the Office Add-ins documentation repo. In this repository, you can find the documentation source files for Office JavaScript API concepts, quick starts, tutorials, and how-to guides. For the best experience, we recommend you view this content on the [Microsoft Docs](https://learn.microsoft.com/office/dev/add-ins) site.
+Welcome to the Office Add-ins documentation repo. In this repository, you can find the documentation source files for Office JavaScript API concepts, quickstarts, tutorials, and how-to guides. For the best experience, we recommend you view the [Office Add-ins documentation](https://learn.microsoft.com/office/dev/add-ins) on Microsoft Learn.
```

Comment:
```suggestion
Welcome to the Office Add-ins documentation repo. In this repository, you can find the documentation source files for Office JavaScript API concepts, quick starts, tutorials, and how-to guides. For the best experience, we recommend you view the [Office Add-ins documentation](https://learn.microsoft.com/office/dev/add-ins) on Microsoft Learn.
```

Reply:
Style guide recommendation is one word for "quickstart".
https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=28014

Reply:
That's not the style we're currently using in our Office docs.

Reply:
Office Style Guide uses "quick start": https://styleguides.azurewebsites.net/StyleGuide/Read?id=2665